### DISCUSSION TITLE
Make backup files optional.

### DISCUSSION THREAD

**Discussion by IdiotBlueMage** [btimofeev/UniPatcher#50] [Ideas]
There's just one thing that keeps UniPatcher from being my main patching app for the platform, and that is that it must always make a new patched file.  There is no way to overwrite a file when saving.  This is especially annoying when applying multiple patches to one file, as with even just two you have backups of backups.

**Comment by btimofeev** [comment]
There were no backups in the first versions of the application. And users just messed up their files when they used the patch on the wrong ROM. But I'll think about your suggestion, it might be worth adding this option to the settings.
